packages/amplify-graphql-api-construct-tests/src/commands.ts

```diff
@@ -256,12 +213,10 @@ export const createGen1ProjectForMigration = async (
  */
 export const deleteDDBTables = async (tableNames: string[]): Promise<void> => {
   const client = new DynamoDBClient({ region: process.env.CLI_REGION || 'us-west-2' });
-  // TODO: GEN1_GEN2_MIGRATION
-  // disable deletion protection before deleting the tables
-  // start block
+  // deletion protection is enabled for migrated tables
+  // disable deletion protection to teardown the tests
   await Promise.allSettled(
```

Comment:
We actually need this block because the tables will still have deletion protection when tearing down the test.